server/views/account/email-update.jade

```diff
@@ -0,0 +1,14 @@
+extends ../layout
+block content
+  h2.text-center Update your email address here:
+  form.form-horizontal.email-update(method='POST', action='/api/users/#{user.id}/email-update', name="updateEmailForm")
```

Comment:
`/api/users/#{user.id}/email-update`

This MUST be changed or anyone can simply call and change anyone else's data via a brute-force.